org/mozilla/jss/tests/TestBufferPRFD.java

```diff
@@ -93,19 +95,23 @@ public synchronized static void TestSSLVersionGetSet(PRFDProxy s_nspr) throws Ex
         assert(SSLVersion.TLS_1_2.value() <= actual.getMaxEnum());
     }
 
-    public synchronized static void TestSSLHandshake(String database, String password) throws Exception {
-        /* Constants */
-        String host = "localhost";
-        byte[] peer_info = host.getBytes();
-
-        /* Find SSL Certificate */
+    public static void InitializeCM(String database, String password) throws Exception {
         CryptoManager manager;
         CryptoManager.initialize(database);
         manager = CryptoManager.getInstance();
         manager.setPasswordCallback(new Password(password.toCharArray()));
         CryptoToken token = manager.getInternalKeyStorageToken();
```

Comment:
Probably an existing issue. The `token` object doesn't seem to be used.

Reply:
ACK, I can drop that.